APIAssignment1.py

```diff
@@ -0,0 +1,101 @@
+import json
+import requests
+import csv
+
+# 1. Write a function that would provide the below mentioned rocket and payload details
+# for the latest historical event.
+# Rocket name
+# Rocket height and diameter in meters
+# Payload id
+# Payload nationality and mass in kg
+def rocketdetails():
+    info = [] #stores attributes for output, will write to csv later
```

Comment:
I see at least two conventions on the same file, `#xyz` and `# xyz`, there's no right or wrong, but at least maintain the same convention (spacing, casing)